src/plugins/typechecker/simpleTestSpecification.ini

```diff
@@ -0,0 +1,85 @@
+#@META fallback/#1 = spec/examples/intKey
+fallbackIntKey = 
+
+#@META check/evenInt = _
+#@META check/int = _
+#@META check/positiveInt = _
+#@META restrict/null = _
+#@META restrict/write = _
+intKey = 5
+
+#@META check/evenInt = _
+#@META check/int = _
+#@META check/positiveInt = _
+#@META restrict/null = _
+#@META restrict/write = _
+otherIntKey = 6
+
+#@META check/int = _
+#@META fallback/#1 = spec/examples/otherIntKey
+otherFallbackIntKey = 
+
+#@META restrict/write = _
+restricted = 5
+
+#@META override/#1 = spec/examples/restricted
+overrideRestricted = 5
+
+nullary = 5
+
+#@META override/#1 = spec/examples/nullary
+#@META restrict/null = _
+nonNullarry = 6
+
+#@META check/evenInt = _
+#@META fallback/#1 = spec/examples/intKey
+otherKey = 
+
+untypedKey = 
+
+#@META fallback/#1 = spec/examples/otherKey
+#@META fallback/#2 = spec/examples/untypedKey
+transitiveFallbackKey = 
+
+#@META fallback/#1 = spec/examples/fallbackIntKey
+#@META override/#1 = spec/examples/untypedKey
+overwriteKey = 
+
+#@META check/evenInt = _
+#@META check/positiveInt = _
+#@META fallback/#1 = spec/examples/intKey
+incompatibleKey = 
+
+#@META specElektra/type = self :: a -> self :: ('TyLft 'In NonNull a)
```

Comment:
What is TyLft?

Reply:
The current attempt to model non-nullness as a higher order type, but i think this will change in the final specification anyway, so i wouldn't pay too much attention to that for now ;)